src/app.js

```diff
@@ -18,6 +18,9 @@ app.get("/", (req, res) => {
   res.send("Hello, boilerplate!");
 });
 
+// Create user route
+app.use('/api/users', require('./routes/users'));
```

Comment:
@Igor333m to make cleaner, I think moving that require near the top and declare it like so `const userRouter = require(...)` and you would just pass `userRouter` into the `app.use()`